code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -265,6 +265,8 @@
 	return JOB_AVAILABLE
 
 /mob/dead/new_player/proc/AttemptLateSpawn(datum/job/job, obj/structure/overmap/ship/simulated/ship)
+	if (isnull(ship))
```

Comment:
Note to self, check `ship` and `ship.shuttle` and stack trace it to see why this is even happening